tidb-cloud/tidb-cloud-roadmap.md

```diff
@@ -144,15 +144,10 @@ For the roadmap of TiDB kernel, refer to [TiDB Roadmap](https://github.com/pingc
     <td>✅ TiDB Cloud supports replicating data to Kafka/MySQL.</td>
     <td>TiDB Cloud supports TiCDC-based data replication to Kafka and MySQL compatible databases.</td>
   </tr>
-  <tr>
-    <td>Backup and Restore</td>
-    <td>✅ Support EBS snapshot-based backup and restore.</td>
-    <td>BR service on TiDB Cloud uses EBS snapshot-based backup and restore.</td>
-  </tr>
   <tr>
     <td>Backup and restore</td>
-    <td>Backup and restore service based on AWS EBS or GCP persistent disk snapshots.</td>
-    <td>Provide backup and restore service on the cloud based on AWS EBS or GCP persistent disk snapshots.</td>
+    <td>Backup and restore performance using optimized file based backup.</td>
```

Comment:
Hi [benmeadowcroft](https://github.com/benmeadowcroft), as the latest TiDB Cloud source files are maintained in the release-6.5 branch for now, would you please create another PR for this roadmap change instead?

Reply:
Yes, sorry this was a mistake, have reverted the commit